pkg/setup/setup.go

```diff
@@ -77,9 +77,9 @@ BUCKET_ROOT_DIR=%s
 TARGET_UTIL=%d
 
 adduser qualifier
-cd /home/qualifier
+cd /home/qualifier || :
```

Comment:
Was this something shellcheck nagged about? I'm not sure if userdata scripts are executed with bash strict mode or not, but we should probably do a `set -euo pipefail` at the top anyways. In the future, we should move this outside the go code.

Reply:
I 100% agree with everything you said. If it didn't take me nearly all day just to get e2e tests passing i'd address it in this PR, but I don't think my computer can take anymore verbal abuse today.

* Yes, shellcheck nagged and I initially resolved with `set -euo pipefail` , but then e2e tests started failing. Replacing with ` || :` resolved the nag and e2e tests pass. I need to deep dive this more tomorrow.

I'm okay keeping the PR open until I can get shellcheck resolved correctly. I'll address moving this out of `setup.go` in the next PR